ci/jenkins/test-vmc.sh

```diff
@@ -353,6 +353,13 @@ function deliver_antrea {
     # Clean up dangling images generated in previous builds. Recent ones must be excluded
     # because they might be being used in other builds running simultaneously.
     docker image prune -f --filter "until=1h" > /dev/null
+    free_space=$(df -h / | awk 'NR==2 {print $4}' | sed 's/G//')
```

Comment:
If there is not much space left, there may not be a "G" unit. Also on my system, the unit is "Gi", not "G"...

Reply:
Maybe we could use `df -h -B 1G`?